app/models/user.rb

```diff
@@ -91,4 +94,8 @@ def project_role_for(project)
   def set_token
     self.token = SecureRandom.hex
   end
+
+  def set_time_format
+    self.time_format = 'relative'
```

Comment:
can this be the column default -> remove method + allow initializing with a different format